packages/angular/tslint.json

```diff
@@ -0,0 +1,140 @@
+{
+  "extends": "tslint:recommended",
```

Comment:
Could we use an `airbnb` compatible linter for that?

Reply:
I've migrated TSLint to ESLint, next step, try to use airbnb

Reply:
Added also AirBnB, not sure if it's the best, but there is a try :)